tests/acceptance/features/bootstrap/Sharing.php

```diff
@@ -2005,15 +2005,20 @@ public function theUserHasDeletedLastShareUsingTheSharingAPI():void {
 	/**
 	 * @param string $user
 	 * @param string|null $sharer
+	 * @param bool $deleteLastPublicLink
 	 *
 	 * @return void
 	 */
-	public function deleteLastShareUsingSharingApi(string $user, string $sharer = null):void {
+	public function deleteLastShareUsingSharingApi(string $user, string $sharer = null, bool $deleteLastPublicLink=false):void {
```

Comment:
```suggestion
	public function deleteLastShareUsingSharingApi(string $user, string $sharer = null, bool $deleteLastPublicLink = false):void {
```

will `test-php-style` complain about missing spaces?

Reply:
`test-php-style` do not complain and `test-php-style-fix` did not fix that.